src/tir/analysis/calculate_workspace.cc

```diff
@@ -33,10 +33,12 @@ class WorkspaceCalculator : public StmtExprVisitor {
  public:
   WorkspaceCalculator() = default;
   size_t operator()(const PrimFunc& func);
+  size_t byte_alignment = 1;
```

Comment:
maybe this should become a constant somewhere just to make sure we use the correct default always. this isn't ideal--if Target was the correct place for this I'd just suggest we make it the default there. but perhaps this can be a stopgap til we fix Target

Reply:
How about kWorkspaceAllocAlignment in include/tvm/runtime/device_api.h ?

Reply:
Done!